test/test_facevalues.jl

```diff
@@ -199,3 +199,30 @@
     end
 
 end # of testset
+
+@testset "EmbeddedLineFacetValues" begin
+    lₕ = 4.0
+    lᵥ = 3.0
+    l = sqrt(lₕ^2 + lᵥ^2)
+
+    for (order, ct) in zip((1, 2), (Line, QuadraticLine))
+        grid = generate_grid(ct, (1,), Vec((0.0, 0.0)), Vec((lₕ, lᵥ)))
+        ip = Lagrange{RefLine, order}()
+        ipGeo = Lagrange{RefLine, order}()^2
+
+        fqr = FacetQuadratureRule{RefLine}(1)
+        fv = FacetValues(fqr, ip, ipGeo)
+        fc = FacetCache(grid)
+
+        # The normals are orthogonal to the line
```

Comment:
This is wrong - the normals should be parallel/antiparallel to the line (pointing out from the cell). 

The best way to test this IMO would be to check that 
```math
\boldsymbol{n} = \frac{\boldsymbol{v}}{||\boldsymbol{v}||}, \quad \boldsymbol{v} = s \frac{\partial\boldsymbol{x}}{\partial \xi_1}
```
where ``s`` is -1 for $\xi_1=-1$ and +1 for $\xi_1=1$ by using a numerical derivative. 

The good thing is that this should become simply
```julia
function weighted_normal(J::SMatrix{dim, 1, T}, ::Type{RefLine}, facet::Int) where {dim, T}
    (facet < 1 || facet > 2) && throw(ArgumentError("unknown facet number"))
    s = facet == 1 ? -1 : 1
    return s * Vec{dim, T}((J...))
end
```
and would then also work for 3d.